libraries/AP_HAL_ChibiOS/hwdef/BOTWINGF405/hwdef.dat

```diff
@@ -0,0 +1,220 @@
+# hw definition file for BOTWING-F405 hardware
+# tested on the BOTWING-F405 board
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_BOTWINGF405
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+define STM32_ST_USE_TIMER 5
+define CH_CFG_ST_RESOLUTION 32
+
+# reserve 16k for bootloader, 16k for OSD and 32k for flash storage
+FLASH_RESERVE_START_KB 64
```

Comment:
Are you only using this on copter? Many of our boards use 48k to save flash for bigger builds.

Reply:
You will need to redo the bootloader as well